client/js/templates/Item.jsx

```diff
@@ -360,15 +394,30 @@ export default function Item({ currentTime, item, selected, expanded, setNavExpa
     }, [configuration, expanded, item.id, item.unread, previouslyExpanded]);
 
     const entryOnClick = useCallback(
-        (event) => handleClick({ event, history, location, expanded, id: item.id, target: '.entry' }),
+        (event) => handleToggleOpenClick({ event, history, location, expanded, id: item.id, target: '.entry' }),
         [history, location, expanded, item.id]
     );
 
     const titleOnClick = useCallback(
-        (event) => handleClick({ event, history, location, expanded, id: item.id, target: '.entry-title' }),
+        (event) => handleToggleOpenClick({ event, history, location, expanded, id: item.id, target: '.entry-title' }),
         [history, location, expanded, item.id]
     );
 
+    const titleOnMultiClick = useMultiClickHandler({
+        0: (event) => {
+            event.preventDefault();
+        },
+        1: titleOnClick,
+        2: useCallback(
```

Comment:
`useCallback` would be needed if `handler[2]` was used in dependencies list of `useEffect` but in this case, the whole `handler` object is used so that needs to be wrapped in `useMemo` instead, see https://react.dev/reference/react/useMemo#memoizing-a-dependency-of-another-hook

Reply:
I probably miss a lot of React's basic rendering concept to truly understand that :see_no_evil: So, please excuse if I don't understand it right away, but from your info and the docs, combined with my limited understanding, it should look like the following? :thinking: 

```javascript
    const titleOnMultiClick = useMultiClickHandler(useMemo(() => {
        0: (event) => {
            event.preventDefault();
        },
        1: (event) => handleToggleOpenClick({ event, history, location, expanded, id: item.id, target: '.entry-title' }),
        2: (event) => {
            if (canWrite && !selfoss.isSmartphone()) {
                handleToggleReadClick({ event, unread: item.unread, id: item.id });
            }
        )
    }, [ canWrite, history, location, expanded, item.unread, item.id ]));
```

Reply:
Yes, that looks correct to me.